sections/variant-list.liquid

```diff
@@ -0,0 +1,182 @@
+    <div class="variant__items" id="main-variant-items">
+      <div class="js-contents">
+        <table class="variant-items">
+          <caption class="visually-hidden">
+            {{ 'sections.cart.title' | t }}
+          </caption>
+          <thead>
+            <tr>
+              <th class="caption-with-letter-spacing" colspan="1" scope="col">
+                {%- if product.has_only_default_variant -%}
+                  {{ 'sections.variant_list.product' | t }}
+                {%- else -%}
+                  {{ 'sections.variant_list.variant' | t }}
+                {%- endif -%}
+              </th>
+              <th class="medium-hide large-up-hide right caption-with-letter-spacing" colspan="1" scope="col">
+                {%- if product.has_only_default_variant -%}
+                  {{ 'sections.variant_list.product_total' | t }}
+                {%- else -%}
+                  {{ 'sections.variant_list.variant_total' | t }}
+                {%- endif -%}
+              </th>
+              <th class="variant-items__heading--wide small-hide caption-with-letter-spacing" colspan="1" scope="col">
+                {{ 'products.product.quantity.label' | t }}
+              </th>
+              <th class="variant-items__heading--wide small-hide caption-with-letter-spacing" colspan="1" scope="col">
+                {{ 'sections.cart.headings.price' | t }}
+              </th>
+              <th class="small-hide right caption-with-letter-spacing" colspan="1" scope="col">
+                {%- if product.has_only_default_variant -%}
+                  {{ 'sections.variant_list.product_total' | t }}
+                {%- else -%}
+                  {{ 'sections.variant_list.variant_total' | t }}
+                {%- endif -%}
+              </th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {%- if product.has_only_default_variant -%}
+              {%- render 'variant-list-row',
+                item: product,
+                image: product.featured_media,
+                sku: product.selected_or_first_available_variant.sku,
+                variant: product.selected_or_first_available_variant
+              -%}
+            {%- else -%}
+              {%- for variant in product.variants -%}
+                {%- render 'variant-list-row',
+                  item: variant,
+                  image: variant.image,
+                  sku: variant.sku,
+                  variant: variant
+                -%}
+              {%- endfor -%}
+            {%- endif -%}
+          </tbody>
+        </table>
+      </div>
+    </div>
+
+    <p class="visually-hidden" id="cart-live-region-text" aria-live="polite" role="status"></p>
+    <p
+      class="visually-hidden"
+      id="shopping-cart-variant-item-status"
+      aria-live="polite"
+      aria-hidden="true"
+      role="status"
+    >
+      {{ 'accessibility.loading' | t }}
+    </p>
+  </form>
+
+  {%- unless product.has_only_default_variant -%}
```

Comment:
<img src="https://screenshot.click/09-24-yjl2x-qlxe5.png" width="400" />

Totals element doesn't receive gradient and also doesn't appear full width as the rest of the variant list is which is noticeable when you use a gradient color scheme on the section.

Reply:
Since this is related to the sticky bar, ill address it in https://github.com/Shopify/dawn/pull/2690